composer.json

```diff
@@ -32,7 +32,7 @@
         "symfony/security-bundle": "^4.3 || ^5.0 || ^6.0 || ^7.0",
         "symfony/twig-bundle": "^4.3 || ^5.0 || ^6.0 || ^7.0",
         "symfony/validator": "^4.3 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/phpunit-bridge": "^5.0.4 || ^6.0 || ^7.0",
+        "symfony/phpunit-bridge": "^5.0.4 || ^6.0 || 7.1.*",
```

Comment:
The 7.2 bridge currently fails with: PHP Fatal error:  Uncaught Error: Call to private PHPUnit\Framework\TestSuite::__construct() from scope PHPUnit\Util\Configuration in /home/runner/work/MassiveSearchBundle/MassiveSearchBundle/vendor/bin/.phpunit/phpunit-8-1/src/Util/Configuration.php:990

Reply:
Looks like this is failing since a longer time without a proper fix yet. So I will ignore the issue.